docs/guides/using-requirements-file.rst

```diff
@@ -0,0 +1,117 @@
+Using a Requirements File
+=========================
+
+The requirements file option is useful in the following scenarios:
+ - You are using external packages, extensions, and themes that you need to include in your documentation project
+ - You have enabled the install project in a virtual environment option, but your package's setup.py file is not in the root directory.
+ - You have multiple python packages in your project and you wish to document them.
+
+-----------------------------
+Using a pip requirements file
+-----------------------------
+
+Creating the requirements file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You are using external packages, extensions, and themes that you need to include in your documentation project
```

Comment:
This heading is very long and probably should probably be shorter. I realize the idea was to match the 3 cases outlined in the first section but perhaps something like "Supporting external packages, extensions, and themes" would work.